src/styles/_catalog.scss

```diff
@@ -1,6 +1,12 @@
+$card-width: 200px;
+$grid-gap: 48px;
```

Comment:
I think that variable for gap isn`t necessary here because you will use it only once.
```suggestion
```